pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -114,6 +114,15 @@ var (
 
 	audience = env.RegisterStringVar("AUDIENCE", "",
 		"Expected audience in the tokens. ")
+
+	defaultTLSServerCertChain = env.RegisterStringVar("TLS_SERVER_CERT_CHAIN", "",
+		"Default location for file based server x509 certificate.").Get()
+
+	defaultTLSServerKey = env.RegisterStringVar("TLS_SERVER_KEY", "",
+		"Default location for file based server x509 private key.").Get()
+
+	defaultTLSServerRootCert = env.RegisterStringVar("TLS_SERVER_ROOT_CERT", "",
+		"Default location for file based server x509 root certificate.").Get()
```

Comment:
Is certcontroller.go a better place for these defaults?

Reply:
I don't think it has anything to do with istio_ca - and probably not with certcontroller either.

If you intent is to use existing, user-provided certs for the TLS servers ( all of them should use the same cert ) - probably in server.go ?